doc/concept/overview.md

```diff
@@ -0,0 +1,51 @@
+
+### GO Guarantees of Origin
+The existing system is based on Guarantees of Origin (GO) which are issued yearly and are based on the energy produced in a year. This system has been in place for a long time and is well understood. The system is based on the following principles:
+
+1. The energy produced is certified by a third party issuer compliant with a central authority (AIB), and the certificate is issued to the producer
+2. The certificate is then sold to the supplier
+3. The supplier then sells the certificate to the consumer
+4. The consumer can then use the certificate to claim/retire/cancel that the energy supplied is renewable
+
+The existing GO system is based on the following principles:
+- Yearly certification
+- Volume based accounting
+- Exchange of certificates between parties through a centralised system AIB-hub
+- The certificate is a tradable asset
+- No direct physical link between the certificate and the energy supplied are required
+
+With the penetration of renewable energy increasing, at times more than 100% of certain grid areas are supplied by renewables, it brings a number of challenges to the existing system: 
+- Makes it hard to balance the energy grid to 50Hz
+- Creates bottlenecks in the existing grid 
+- Requires loadbalancing that is less centralized 
+- Makes it hard to validate the actual energy supplied to power-to-x applications and the origin of the losses in the process
+
+Guarantees of Origin are based on a standard developed by the European Committee for Standardisation (CEN) and the European Committee for Electrotechnical Standardisation (CENELEC) and is called EN 16325. [EECS and the CEN standard EN16325](https://www.aib-net.org/eecs) AIB is the central authority for the GO system in Europe and is responsible for the certification of the issuers and the central hub for the exchange of certificates. [AIB](https://www.aib-net.org/) The regulation is based on the Renewable Energy Directive (RED) and the [REDII - regulation](https://energy.ec.europa.eu/topics/renewable-energy/renewable-energy-directive-targets-and-rules/renewable-energy-directive_en).
+
+It is by no means an extensive and exhaustive analysis of the existing Guarantees of Origin system, but it is a good starting point for understanding the system. Tackling the challenges of the existing system is the basis for the Energy Track and Trace project and a strategy for tackling the challenges is described in the next section.
+
+### Strategies that could be applied to the problem
+Several strategies could be applied to the infrastructure to tackle the challenges of the existing system. 
```

Comment:
This section taps more into the "how?", while the above is more of the "what?" and "why?" - Hence, we are reaching aspects, where my contributions may be limited